### DISCUSSION TITLE
useForm 的 autoComplete怎么禁用

### DISCUSSION THREAD

**Discussion by Angry-Sparrow** [ant-design/ant-design#39659] [Q&A] [answered]
使用useForm时，表单总有一个自动options，怎么禁用掉呢
<img width="435" alt="image" src="https://user-images.githubusercontent.com/48384080/208426105-ef9560b1-aa3c-4c23-adf2-cd3e71981b3e.png">

https://stackblitz.com/edit/react-wrrkto?file=demo.tsx

**Comment by zombieJ** [comment, marked as answer]
Form 有 `autoComplete` 属性，可以关闭

**Reply by Angry-Sparrow** [reply]
> Form 有 `autoComplete` 属性，可以关闭

试过了，不生效。
<img width="994" alt="image" src="https://user-images.githubusercontent.com/48384080/208568860-498c5f3e-c73b-4d3c-8d1f-7812ea8a6684.png">

官方demo的版本是：
<img width="321" alt="image" src="https://user-images.githubusercontent.com/48384080/208568921-7bd52a32-e04a-40b8-a9bf-84839d9f1627.png">
5.0.7

我本地使用的版本是：4.23.6

**Reply by Angry-Sparrow** [reply]
看到源码得写 autoComplete="off" 解决了。
